base/exports.jl

```diff
@@ -1180,8 +1180,10 @@ export
 # multiprocessing
     addprocs,
     asyncmap,
+    CachingPool,
     ClusterManager,
     default_worker_pool,
+    define,
```

Comment:
Agree with @tkelman ; if we have this function it should be called something like `remoteset!`. Modifying global variables, especially in other processes, is not the kind of thing that gets a nice name.